src/main/java/net/authorize/sample/TransactionReporting/GetAccountUpdaterJobDetails.java

```diff
@@ -0,0 +1,177 @@
+import net.authorize.api.contract.v1.GetAUJobDetailsRequest;
+import net.authorize.api.contract.v1.GetAUJobDetailsResponse;
+import net.authorize.api.contract.v1.MerchantAuthenticationType;
+import net.authorize.api.contract.v1.MessageTypeEnum;
+import net.authorize.api.contract.v1.Paging;
+import net.authorize.api.controller.GetAUJobDetailsController;
+import net.authorize.api.controller.base.ApiOperationBase;
+
+public class GetAccountUpdaterJobDetails {
+
+	public static ANetApiResponse run(String apiLoginId, String transactionKey) {
+
+		// Set the request to operate in either the sandbox or production environment
+		ApiOperationBase.setEnvironment(Environment.SANDBOX);
+
+		// Create object with merchant authentication details
+		MerchantAuthenticationType merchantAuthenticationType = new MerchantAuthenticationType();
+		merchantAuthenticationType.setName(apiLoginId);
+		merchantAuthenticationType.setTransactionKey(transactionKey);
+
+		String month = "2018-08";
+		String reFid = "123456";
+		Paging paging = new Paging();
+		paging.setLimit(100);
+		paging.setOffset(2);
+
+		// Create the API request and set the parameters for this specific request
+		GetAUJobDetailsRequest apiRequest = new GetAUJobDetailsRequest();
+		apiRequest.setMerchantAuthentication(merchantAuthenticationType);
+		apiRequest.setPaging(paging);
+		apiRequest.setRefId(reFid);
+		apiRequest.setMonth(month);
+		apiRequest.setModifiedTypeFilter(AUJobTypeEnum.ALL);
+
+		// Call the controller
+		GetAUJobDetailsController controller = new GetAUJobDetailsController(apiRequest);
+		controller.execute();
+
+		GetAUJobDetailsResponse response = new GetAUJobDetailsResponse();
+		response = controller.getApiResponse();
+
+		// If API Response is OK, go ahead and check the transaction response
+		if (response != null && response.getMessages().getResultCode() == MessageTypeEnum.OK) {
+
+			System.out.println("SUCCESS: Get Account Updater job details for Month and year : " + month);
+
+			if (response.getAuDetails() == null) {
+				System.out.println("No GetAccountUpdaterjobdetails for this month and year.");
+				return response;
+			}
+
+			ArrayList<AuUpdateType> updateTypeList = new ArrayList<AuUpdateType>();
+			ArrayList<AuDeleteType> deleteTypeList = new ArrayList<AuDeleteType>();
+
+			for (AuDetailsType details : response.getAuDetails().getAuUpdateOrAuDelete()) {
+
+				System.out.println("---Customer profile details Start---");
+				System.out.println("customerProfileID:" + details.getCustomerProfileID());
+				System.out.println("customerPaymentProfileID:" + details.getCustomerPaymentProfileID());
+				System.out.println("customerFirstname:" + details.getFirstName());
+				System.out.println("customerLastname:" + details.getLastName());
+				System.out.println("updateTimeUTC:" + details.getUpdateTimeUTC());
+				System.out.println("reasonCode:" + details.getAuReasonCode());
+				System.out.println("reasonDescription:" + details.getReasonDescription());
```

Comment:
Please check the Java sample codes for the strings to print here. For example, it should print "Profile ID", and NOT "customerProfileID".

Reply:
Changes done as per suggestion.